defines/defines.go

```diff
@@ -35,6 +35,7 @@ const (
 	BatchIntervalUploadStats        = 0.95
 	ChannelInBufferSize             = 800
 	ChannelOutBufferSize            = 800
+	DefaultConfigFile               = "/etc/arachne/arachne.yaml"
```

Comment:
Actually, this const is not even needed since the default is already provided by the `default` CLI option.